src/msg/MessageRef.h

```diff
@@ -21,9 +21,12 @@ template<typename T>
 using MRef = boost::intrusive_ptr<T>;
 template<typename T>
 using MConstRef = boost::intrusive_ptr<T const>;
+template<typename T, typename D>
+using MURef = std::unique_ptr<T, D>;
```

Comment:
Can we get rid of template `D` as all the `MURef` will need the same type of deleter?
```
template<typename T>
using MURef = std::unique_ptr<T, decltype(&TOPNSPC::common::unique_ptr_release)>;
```